sim/midas/src/main/cc/bridges/fased_memory_timing_model.h

```diff
@@ -3,16 +3,8 @@
 #ifndef __FASED_MEMORY_TIMING_MODEL_H
 #define __FASED_MEMORY_TIMING_MODEL_H
 
-/* This is the widget driver for FASED memory-timing models
- *
- * FASED instances are FPGA-hosted and only rely on this driver to:
- * 1) set runtime-configurable timing parameters before simulation commences
- * 2) poll instrumentation registers
- *
- */
-
 #include "bridges/fpga_model.h"
```

Comment:
Can you not delete fpga_model now?

Reply:
The other classes still use it.

Reply:
Why do they still need to use it?

Reply:
I don't want to include further changes into this refactoring. People more involved in the bridge can subsequently remove the FpgaModel.

Reply:
I seem to have infected subclasses of FASED with `fpga_model`. As part of this PR i do think it make sense to maybe migrate the contents of the file into `FASEDMemoryTImingModel.h` so that it's not accidentally used by someone else. That should be a minor change.